draft-ietf-ice-pac.xml

```diff
@@ -191,15 +191,17 @@
       </t>
       <t>
         While the timer is running, the ICE agent MUST NOT set the state of
-        a checklist to Failed, even if the checklist has no pairs left to check.
-        As a result, the ICE agent will not remove any data streams, or set the state 
-        of the ICE session to Failed, while the timer is till running, allowing
-        received peer-reflexive candidates to be tested. 
+        a checklist to Failed, even if the checklist has no pairs left to check,
+        and MUST wait for the timer to elapse before doing so.
+        As a result, the ICE agent will not remove any data streams or set the
+        state of the ICE session to Failed until the timer elapses.
```

Comment:
I think saying "MUST NOT do X while the timer is running, and MUST wait until the timer has elapsed before doing X" sounds strange. You say the same thing twice. If you really want to keep both phrases, perhaps use two sentences:

"MUST NOT do X while the timer is running. Instead it MUST wait until the timer has elapsed before doing X"

Reply:
I agree it's probably no longer needed now that we have the para below that talks about what to do when the timer fires. I'll adjust accordingly.

Reply:
PTAL

Reply:
Looks good.